adapter/internal/operator/synchronizer/data_store.go

```diff
@@ -148,6 +149,12 @@ func updateHTTPRoute(httpRoute *HTTPRouteState, cachedHTTPRoute *HTTPRouteState,
 			}
 		}
 	}
+
+	if !reflect.DeepEqual(cachedHTTPRoute.BackendPropertyMapping, httpRoute.BackendPropertyMapping) {
```

Comment:
don't we have to check generation?

Reply:
In `Authentication`, there is a generation number since it's a k8s resource but here `BackendPropertyMapping` is a structure we defined in out code. Due to that we need some other way to detect the change as here